src/main/kotlin/kitchenpos/tobe/eatinorder/domain/EatInOrderCompleteService.kt

```diff
@@ -0,0 +1,31 @@
+package kitchenpos.tobe.eatinorder.domain
+
+import kitchenpos.order.common.domain.OrderStatus
+import kitchenpos.tobe.eatinorder.domain.repository.EatInOrderRepository
+import kitchenpos.tobe.support.DomainService
+import java.util.*
+
+@DomainService
+class EatInOrderCompleteService(
```

Comment:
중요한 부분은 아니지만 응용 서비스와 구분을 위해 아래와 같이 하기도 해요.

```suggestion
class EatInOrderCompleteDomainService(
```

Reply:
서비스 이름부터 명시적으로 도메인과 관련되어있음을 나타내는것이군요~ 수정해두겠습니다.